Rosyk_Task1.py

```diff
@@ -5,4 +5,5 @@
         print("Владислав Валерійович Овсяніков")
```

Comment:
It looks like you somehow pushed the code above outside the PR.
please, don't do it anymore. any code that you are going to add should be reviewed

Reply:
if you have start parameter as '0' for your **range**, just skip it. It's default value